site/content/docs/cli/constructs.md

```diff
@@ -371,7 +371,7 @@ const group = new CheckGroup('check-group-1', {
 
 new ApiCheck('check-group-api-check-1', {
   name: 'API check #1',
-  group,
+  group: checkGroup,
```

Comment:
@checklyalex this is valid ts as far as I know. What are you looking to achieve here?